### PR TITLE
Create merge script

### PR DESCRIPTION
**Pull request by Xodarap**
I always have to spend 2 minutes reminding myself of the syntax for merging users, so I just created a script to do it.

I also fixed a typo in the EA forum's spam notification text.

### CODE REVIEW

**Comment by jpaddison3** [comment]
Oh my god I love it. (Will review later)

**Comment by darkruby501** [comment]
Nice!!

File: scripts/mergeUsers.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   echo "h         Print this Help."
+   echo "sourceId  User id of the source user (the one which will get deleted and merged into the target user)."
+   echo "targetId  User id of the target user."
+   echo
+   echo "Example: scripts/mergeUsers.sh xxCmncgCqPxyymEKW QQyWAsWK7PggCXttz"
+}
+
+while getopts "h" flag; do
+case "$flag" in
+    h)
```
Comment (jpaddison3) [inline scripts/mergeUsers.sh:18]: WTF is bash syntax even

File: scripts/mergeUsers.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Help
+    exit 1
+fi
+
+SOURCE_ID=${@:$OPTIND:1}
+TARGET_ID=${@:$OPTIND+1:1}
+
+echo "Merging user $SOURCE_ID into $TARGET_ID"
+
+scripts/serverShellCommand.sh "Vulcan.mergeAccounts('$SOURCE_ID', '$TARGET_ID')"
```
Comment (jpaddison3) [inline scripts/mergeUsers.sh:36]: Files end in newlines

**Review by jpaddison3 — APPROVED** [review]
LGTM. Though I propose we prefer writing scripts like this in typescript, because bash is really a terrible language.

**Pull request merged**